### ISSUE TITLE
Cannot build release changelog

### ISSUE THREAD

**Issue by endorama** [ansible-collections/community.sops#74]
@felixfontein while building the changelog for the release I encountered this error:

```
❯ antsibull-changelog release -v
ERROR! vars community.sops.sops missing documentation (or could not parse documentation): unknown doc_fragment(s) in file /tmp/antsibull-changelog075ng27d/collections/ansible_collections/community/sops/plugins/vars/sops.py: ansible.builtin.vars_plugin_staging
Traceback (most recent call last):
  File "/home/endorama/.cache/pypoetry/virtualenvs/community.sops-kPWhnm6H-py3.8/lib/python3.8/site-packages/antsibull_changelog/cli.py", line 228, in run
    return arguments.func(arguments)
  File "/home/endorama/.cache/pypoetry/virtualenvs/community.sops-kPWhnm6H-py3.8/lib/python3.8/site-packages/antsibull_changelog/cli.py", line 435, in command_release
    plugins = load_plugins(paths=paths, collection_details=collection_details,
  File "/home/endorama/.cache/pypoetry/virtualenvs/community.sops-kPWhnm6H-py3.8/lib/python3.8/site-packages/antsibull_changelog/plugins.py", line 400, in load_plugins
    _load_collection_plugins(plugins_data, paths, collection_details, use_ansible_doc)
  File "/home/endorama/.cache/pypoetry/virtualenvs/community.sops-kPWhnm6H-py3.8/lib/python3.8/site-packages/antsibull_changelog/plugins.py", line 350, in _load_collection_plugins
    plugins_data['plugins'][plugin_type] = load_plugin_metadata(
  File "/home/endorama/.cache/pypoetry/virtualenvs/community.sops-kPWhnm6H-py3.8/lib/python3.8/site-packages/antsibull_changelog/plugins.py", line 298, in load_plugin_metadata
    plugins_data = run_ansible_doc(paths, playbook_dir, plugin_type, plugins_list)
  File "/home/endorama/.cache/pypoetry/virtualenvs/community.sops-kPWhnm6H-py3.8/lib/python3.8/site-packages/antsibull_changelog/plugins.py", line 267, in run_ansible_doc
    output = subprocess.check_output(command)
  File "/home/endorama/.asdf/installs/python/3.8.10/lib/python3.8/subprocess.py", line 415, in check_output
    return run(*popenargs, stdout=PIPE, timeout=timeout, check=True,
  File "/home/endorama/.asdf/installs/python/3.8.10/lib/python3.8/subprocess.py", line 516, in run
    raise CalledProcessError(retcode, process.args,
subprocess.CalledProcessError: Command '['ansible-doc', '--json', '-t', 'vars', '--playbook-dir', '/tmp/antsibull-changelog075ng27d', 'community.sops.sops']' returned non-zero exit status 1.
```

The line `unknown doc_fragment(s) in file /tmp/antsibull-changelog075ng27d/collections/ansible_collections/community/sops/plugins/vars/sops.py: ansible.builtin.vars_plugin_staging` point to line 66 in `sops.py` vars plugin:
https://github.com/ansible-collections/community.sops/blob/adaed57b321a16bc41abcd2b88b6af15a329bbfa/plugins/vars/sops.py#L65-L67

Is it ok to remove that line? Has been added in #32 

I'm using `antsibull-changelog` 0.11.0

**Comment by felixfontein** [comment]
Removing that line removes functionality from the plugin. It looks like you are using a too old Ansible version; that doc fragment is available both in ansible-base 2.10 and ansible-core 2.11, I think it's only not available in Ansible 2.9.

**Comment by endorama** [comment]
Indeed! Upgrading to `4.1.0` works. Release `1.1.0` ongoing
